tests/sles4sap/netweaver_install.pm

```diff
@@ -52,6 +52,24 @@ sub run {
     # Mount media
     $self->mount_media($proto, $path, '/sapinst');
 
+    # Workaround for SLE16 if variable WORKAROUND_BSC1234806 set
+    if (get_var("WORKAROUND_BSC1236235")) {
+        record_soft_failure("bsc#1236235: workaround by installing libnsl package from NFS");
+        assert_script_run "rpm -Uvh /mnt/libnsl1-2.38-160000.4.4." . get_var("ARCH") . ".rpm";
```

Comment:
does the version have to be `2.38-160000.4.4`, we can use fuzzy match if not

Reply:
This is a fixed version package, which is located at NFS.